js/src/data/reducer.js

```diff
@@ -46,10 +46,19 @@ function chain( state, basePath = '' ) {
 		}
 		return clone( value );
 	};
+	const combinBasePath = ( path ) => {
+		if ( basePath ) {
+			if ( Array.isArray( basePath ) || Array.isArray( path ) ) {
+				return [].concat( basePath, path );
+			}
+			return `${ basePath }.${ path }`;
+		}
+		return path;
+	};
```

Comment:
💅  This function returns a `string` or an `array` and it took me a while to understand it's because lodash `setWith` 's second parameter `path` can be either a string or an array. Might be better to add a comment here 😃

Reply:
Added by https://github.com/woocommerce/google-listings-and-ads/pull/1184/commits/eb552c08e0536aca8a76718651d47997cafb8ec3.